### PR TITLE
Add output selection support for ethdebug

### PR DESCRIPTION
**Pull request by aarlt**
Depends on #15289.

### CODE REVIEW

**Review by clonker — APPROVED** [review]
Looks good to me. I was wondering if it would also be an option to throw in case the output is requested (together with the appropriate `debug-info` flag) as you're essentially just swallowing it now which might confuse someone unsuspecting. But then again it's not listed in the options so I think it's fine this way as well.

**Comment by aarlt** [comment]
I will still add the standard-json import part to this PR. This part I didn't pushed yet, because I saw that I need to add additional tests to verify validation logic. I will add this later today.

File: solc/CommandLineInterface.cpp [hunk 1]
```diff
@@ -995,6 +996,8 @@ void CommandLineInterface::handleCombinedJSON()
 				contractData[g_strFunDebugRuntime] = StandardCompiler::formatFunctionDebugData(
 					m_assemblyStack->runtimeObject(contractName).functionDebugData
 				);
+			if (m_options.compiler.combinedJsonRequests->ethdebug)
+				contractData[g_strEthdebug] = Json::object();
```
Comment (cameel) [inline solc/CommandLineInterface.cpp:1000]: Combined JSON has been deprecated for ages now (#10278) and we're likely to drop it in the next breaking release. It does not make sense add more things to it at this point.

This should be standalone compiler output instead.

**Comment by aarlt** [comment]
Replaced by  #15289.

**Pull request closed without merging**